### PR TITLE
Completed prep-html-skeleton exercise :D - Andrew Almeida

### PR DESCRIPTION
**Pull request by A-MAlmeida**
5B5F87EC-0372-467B-A4E9-4A9C2EE9D827

### CODE REVIEW

File: prep-html-skeleton/index.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title> HTML Skeleton Exercise </title>
+      <body>
+
+        <h1>I built a proper HTML skeleton!</h1>
+
+      </body>
+  </head>
```
Comment (thebearingedge) [inline prep-html-skeleton/index.html]: Hi Andrew, be sure to divide your page into two specific sections. The `<head>` should open and close be fore the `<body>` opens and closes. The visible content of the page should go in the `<body>`.

```html
<!DOCTYPE html>
<html>
  <head>
    <!-- meta info like <title></title> goes in here -->
  </head>
  <body>
    <!-- visible content like <h1></h1> and <p></p> goes in here -->
  </body>
</html>
```

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by A-MAlmeida** [comment]
C40F156F-297F-4925-BB88-F3B20EDEE3C6

**Comment by A-MAlmeida** [comment]
Do I have to create a new pull request?
I've already pushed the updated assignment through.

thanks for your help!

**Review by thebearingedge — APPROVED** [review]

**Comment by thebearingedge** [comment]
All good!

**Comment by A-MAlmeida** [comment]
YEAHHHHHHH
thank ya

**Pull request merged**